roles/acme/defaults/main.yml

```diff
@@ -17,6 +17,14 @@ acme_fullchain_path: "{{ acme_cert_dir }}/{{ acme_domain.certificate_name }}_ful
 acme_private_key_path: "{{ acme_cert_dir }}/{{ acme_domain.certificate_name }}.key"
 acme_remaining_days: "30"
 
+### subject fields for the resulting cert
+acme_subject_country: "{{ acme_domain.country | default(omit) }}"
```

Comment:
I don't understand where `acme_domain.country` is supposed to come from. These variables don't exist, or do they?

Reply:
The initial values are defined in the ./roles/acme/defaults/main.yml file. If a user does not set these values explicitly via e.g. group_vars or host_vars or as explicit var in his own playbook. The defaults set the values to omit them, so that the community.crypto.openssl_csr does not send these values at all. Let's Encrypt does omit these values too when issuing the certificate. The subject fields just may come in handy when using the ACME protocol via a private PKI.

Reply:
Is there a reason you're not omitting these variables in the task itself?
```
- name: Create CSR for certificate
  openssl_csr:
    path: "{{ acme_csr_path }}"
    privatekey_path: "{{ acme_private_key_path }}"
    email_address: "{{ acme_domain.email_address }}"
    country_name: "{{ acme_domain.country | default(omit) }}"
```

This way we do not have to define a two different names for one variable?

Reply:
You can just update this PR. The commits will get squashed together anyway.

Reply:
Thank you again for the suggestion. I did now change the variable usage in a way that only one variable name will be used in the overall process and the default is to omit the subject fields. 
So, if you still want to, you could merge the pull request now.

Thank you very much
Zephyr82